modules/db_postgres/doc/db_postgres_admin.xml

```diff
@@ -112,6 +112,56 @@ modparam("db_postgres", "max_db_queries", 2)
 ...
 modparam("db_postgres", "timeout", 2)
 ...
+</programlisting>
+		</example>
+	</section>
+
+        <section id="param_use_tls" xreflabel="use_tls">
+		<title><varname>use_tls</varname> (integer)</title>
+		<para>
+		Warning: <emphasis>wolfSSL</emphasis> is the recommended TLS/SSL Library
```

Comment:
This could be expanded a bit, warning about the crashes with openssl and also mentioning that it is possible to use no TLS library for the `tls_mgm` module.